aliyun/src/test/java/org/apache/iceberg/aliyun/AliyunTestUtility.java

```diff
@@ -17,25 +17,31 @@
  * under the License.
  */
 
-package org.apache.iceberg.aliyun.oss;
+package org.apache.iceberg.aliyun;
 
+import org.apache.iceberg.aliyun.oss.AliyunOSSTestRule;
 import org.apache.iceberg.aliyun.oss.mock.AliyunOSSMockRule;
 import org.apache.iceberg.common.DynConstructors;
 import org.apache.iceberg.relocated.com.google.common.base.Strings;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-public class AliyunOSSTestUtility {
-  private static final Logger LOG = LoggerFactory.getLogger(AliyunOSSTestUtility.class);
-  private static final String ALIYUN_TEST_OSS_TEST_RULE_CLASS = "ALIYUN_TEST_OSS_TEST_RULE_CLASS";
+public class AliyunTestUtility {
+  private static final Logger LOG = LoggerFactory.getLogger(AliyunTestUtility.class);
+  private static final String ALIYUN_TEST_OSS_RULE_CLASS = "ALIYUN_TEST_OSS_TEST_RULE_CLASS";
+  private static final String ALIYUN_TEST_OSS_ENDPOINT = "ALIYUN_TEST_OSS_ENDPOINT";
+  private static final String ALIYUN_TEST_ACCESS_KEY_ID = "ALIYUN_TEST_ACCESS_KEY_ID";
+  private static final String ALIYUN_TEST_ACCESS_KEY_SECRET = "ALIYUN_TEST_ACCESS_KEY_SECRET";
+  private static final String ALIYUN_TEST_BUCKET_NAME = "ALIYUN_TEST_BUCKET";
```

Comment:
Nit:  It's better to rename it as `ALIYUN_TEST_OSS_BUCKET_NAME` because the bucket is only used for aliyun oss service.